tests/run_pretrained_models.py

```diff
@@ -170,7 +170,7 @@ def run_caffe2(self, name, model_proto, inputs):
     def run_onnxruntime(self, name, model_proto, inputs):
         """Run test against onnxruntime backend."""
         import onnxruntime as rt
-        model_path = utils.save_onnx_model(TEMP_DIR, name, inputs, model_proto, include_test_data=True)
+        model_path = utils.save_onnx_model(TEMP_DIR, name, inputs, model_proto, include_test_data=True, as_text=True)
```

Comment:
as_text=True when is debug mode?
it will take quite a while to dump pbtxt for large model.

Reply:
done